conf/application.conf

```diff
@@ -21,6 +21,10 @@ application.global=uk.gov.hmrc.agentmappingfrontend.config.FrontendGlobal
 
 play.filters.headers.contentSecurityPolicy= "default-src 'self' 'unsafe-inline' localhost:9000 localhost:9032 www.google-analytics.com data:"
 
+# Whitelisting
+passcodeAuthentication.enabled = true
```

Comment:
I think it would be nice if our services were consistent about whether whitelisting is enabled by default. agent-subscription-frontend has `false` here so I'd recommend changing this to `false` to match.